t/fmt/asciidoc/DelimitedBlocks.adoc

```diff
@@ -182,3 +182,14 @@ __________________________
 
 end of test
 --------------------------
+
```

Comment:
You cannot just add the delimited block to the `.adoc` file and expect the tests to pass... This change needs to be sided by the changes in the `pot`, `po`, `trans` and `norm` files.